tests/console/wicked_before_test.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Do basic checks to make sure system is ready for weaked testing
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils 'systemctl';
+
+sub run {
+
+    select_console('root-console');
+    systemctl('is-active network');
+    systemctl('is-active wicked');
+    assert_script_run('[ -z "$(coredumpctl -1 --no-pager --no-legend)" ]');
+    wait_idle;
+    script_run("curl -L -v " . autoinst_url . "/data/get_interfaces.sh > get_interfaces.sh");
+    script_run("chmod +x ./get_interfaces.sh");
```

Comment:
Both of these should work with `assert_script_run. And better use single ticks by default